### ISSUE TITLE
inital Sync Fails when syncing Windows Network Drive 

### ISSUE THREAD

**Issue by bboule** [owncloud/client#3270]
OS:  CentOS 6

ownCloud:  8.0.2

Client version:  1.8.1

When the user attempts to sync a folder that is a mount of the WND (this is a VERY deep tree) the discovery starts to happen and moves along but eventually is stalls and begins to throw an error message (waiting on screen shot) and the following errors show up in the client side logging:
```
05-20 09:11:06:436 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\lvl3sump\"
05-20 09:11:06:438 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEV 7\MAIN XCU\EASTDR.BT2"
05-20 09:11:06:438 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/TIN/LEV 7/MAIN XCU/EASTDR.BT2"  is not existing
05-20 09:11:06:439 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEV 7\MAIN XCU\EASTDR.BT2"
05-20 09:11:06:439 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\!040_AutoCAD\Gemcom\Bul River\REPORTS\VOLUME_2014.09.24(09h34m53s)_1.rpt"
05-20 09:11:06:440 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\!040_AutoCAD\Gemcom\Bul River\REPORTS\VOLUME_2014.09.24(09h34m53s)_1.rpt"
05-20 09:11:06:442 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\EXTRACT\Cu_1str.MEX"
05-20 09:11:06:442 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/EXTRACT/Cu_1str.MEX"  is not existing
05-20 09:11:06:443 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\EXTRACT\Cu_1str.MEX"
05-20 09:11:06:443 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!010_Management\010_Proposal\04_Final PDF\Old"
05-20 09:11:06:444 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!010_Management\010_Proposal\04_Final PDF\Old"
05-20 09:11:06:446 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Maritz Emails\"
05-20 09:11:06:446 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Maritz Emails"  is not existing
05-20 09:11:06:446 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Maritz Emails\"
05-20 09:11:06:447 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LV5\fw&x-cut\"
05-20 09:11:06:447 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/TIN/LV5/fw&x-cut"  is not existing
05-20 09:11:06:448 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LV5\fw&x-cut\"
05-20 09:11:06:449 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\DARREN\2002\SURPNTS.ASC"
05-20 09:11:06:450 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/DARREN/2002/SURPNTS.ASC"  is not existing
05-20 09:11:06:450 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\DARREN\2002\SURPNTS.ASC"
05-20 09:11:06:451 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEVEL 5\MAIN X-C\END.BT2"
05-20 09:11:06:451 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/TIN/LEVEL 5/MAIN X-C/END.BT2"  is not existing
05-20 09:11:06:452 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEVEL 5\MAIN X-C\END.BT2"
05-20 09:11:06:453 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\polylines.wed"
05-20 09:11:06:453 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/polylines.wed"  is not existing
05-20 09:11:06:454 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\polylines.wed"
05-20 09:11:06:455 0x3feb28 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QLocalSocket*) "C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEV 7\BEFORE\"
05-20 09:11:06:456 0x3feb28 OCC::SocketApi::fileStatus: OO File  "C:/Users/ccscott/ownCloud/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!020_Project_Data/010_SRK/Bul River Minerals/BRM/TIN/LEV 7/BEFORE"  is not existing
05-20 09:11:06:457 0x3feb28 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\ccscott\ownCloud\VAN Projects\01_SITES\Bul_River\1CC046.000_Tailings_Design\!020_Project_Data\010_SRK\Bul River Minerals\BRM\TIN\LEV 7\BEFORE\"
```

Am waiting for sever logs and screen shows of error messages will at to the issue as soon as I receive them.

**Comment by bboule** [comment]
By way of an update, here is a screen cap from the client:

![image](https://cloud.githubusercontent.com/assets/3329523/7772266/2e3fcfac-006b-11e5-9649-713edac9af13.png)


Here is the sync activity:

![image](https://cloud.githubusercontent.com/assets/3329523/7772282/3fd90116-006b-11e5-995c-6c1560fe40e3.png)

**Comment by bboule** [comment]
Per IRC Request here is a log block that precedes the socket API

```
0 09:10:26:026 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/030_Appendices with read_from_db 0
05-20 09:10:26:026 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures [file_id=00184298ocot35xh4t3a]
05-20 09:10:26:027 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures, instruction: INSTRUCTION_NEW <<=
05-20 09:10:26:027 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures"
05-20 09:10:26:027 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures Calling into main thread...
05-20 09:10:26:034 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures"
05-20 09:10:26:968 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures"
05-20 09:10:26:970 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures ...Returned from main thread
05-20 09:10:26:970 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures"
05-20 09:10:26:970 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/040_Figures with read_from_db 0
05-20 09:10:26:970 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers [file_id=00184299ocot35xh4t3a]
05-20 09:10:26:972 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers, instruction: INSTRUCTION_NEW <<=
05-20 09:10:26:972 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers"
05-20 09:10:26:972 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers Calling into main thread...
05-20 09:10:26:986 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers"
05-20 09:10:27:934 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers"
05-20 09:10:27:935 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers ...Returned from main thread
05-20 09:10:27:935 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers"
05-20 09:10:27:936 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/050_Covers with read_from_db 0
05-20 09:10:27:936 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF [file_id=00184300ocot35xh4t3a]
05-20 09:10:27:938 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF, instruction: INSTRUCTION_NEW <<=
05-20 09:10:27:938 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF"
05-20 09:10:27:938 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF Calling into main thread...
05-20 09:10:27:945 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF"
05-20 09:10:28:911 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF"
05-20 09:10:28:912 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF ...Returned from main thread
05-20 09:10:28:912 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF"
05-20 09:10:28:912 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]/060_Final PDF with read_from_db 0
05-20 09:10:28:912 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2]"
05-20 09:10:28:912 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable2] with read_from_db 0
05-20 09:10:28:912 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3] [file_id=00184206ocot35xh4t3a]
05-20 09:10:28:913 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3], instruction: INSTRUCTION_NEW <<=
05-20 09:10:28:913 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3] is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]"
05-20 09:10:28:913 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3] Calling into main thread...
05-20 09:10:28:917 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]"
05-20 09:10:29:856 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 6 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]"
05-20 09:10:29:858 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3] ...Returned from main thread
05-20 09:10:29:858 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text [file_id=00184301ocot35xh4t3a]
05-20 09:10:29:861 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text, instruction: INSTRUCTION_NEW <<=
05-20 09:10:29:861 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text"
05-20 09:10:29:861 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text Calling into main thread...
05-20 09:10:29:873 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text"
05-20 09:10:30:825 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text"
05-20 09:10:30:826 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text ...Returned from main thread
05-20 09:10:30:826 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text"
05-20 09:10:30:826 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/010_Text with read_from_db 0
05-20 09:10:30:826 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables [file_id=00184302ocot35xh4t3a]
05-20 09:10:30:827 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables, instruction: INSTRUCTION_NEW <<=
05-20 09:10:30:827 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables"
05-20 09:10:30:827 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables Calling into main thread...
05-20 09:10:30:838 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables"
05-20 09:10:31:787 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables"
05-20 09:10:31:788 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables ...Returned from main thread
05-20 09:10:31:788 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables"
05-20 09:10:31:788 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/020_Tables with read_from_db 0
05-20 09:10:31:788 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices [file_id=00184303ocot35xh4t3a]
05-20 09:10:31:789 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices, instruction: INSTRUCTION_NEW <<=
05-20 09:10:31:789 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices"
05-20 09:10:31:789 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices Calling into main thread...
05-20 09:10:31:795 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices"
05-20 09:10:32:760 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices"
05-20 09:10:32:763 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices ...Returned from main thread
05-20 09:10:32:763 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices"
05-20 09:10:32:763 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/030_Appendices with read_from_db 0
05-20 09:10:32:763 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures [file_id=00184304ocot35xh4t3a]
05-20 09:10:32:767 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures, instruction: INSTRUCTION_NEW <<=
05-20 09:10:32:767 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures"
05-20 09:10:32:767 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures Calling into main thread...
05-20 09:10:32:787 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures"
05-20 09:10:33:746 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures"
05-20 09:10:33:748 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures ...Returned from main thread
05-20 09:10:33:748 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures"
05-20 09:10:33:748 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/040_Figures with read_from_db 0
05-20 09:10:33:748 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers [file_id=00184305ocot35xh4t3a]
05-20 09:10:33:751 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers, instruction: INSTRUCTION_NEW <<=
05-20 09:10:33:751 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers"
05-20 09:10:33:751 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers Calling into main thread...
05-20 09:10:33:768 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers"
05-20 09:10:34:383 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::CheckQuotaJob created for "https://files.srk.com/owncloud" + "/"
05-20 09:10:34:862 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobResultSlot: void OCC::DiscoveryMainThread::singleDirectoryJobResultSlot(const QList<OCC::FileStatPointer>&) Have 0 results for  "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers"
05-20 09:10:34:863 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers ...Returned from main thread
05-20 09:10:34:863 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers"
05-20 09:10:34:863 0xb0047a0 csync_ftw:  <= Closing walk for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/050_Covers with read_from_db 0
05-20 09:10:34:864 0xb0047a0 csync_walker: directory: ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF [file_id=00184306ocot35xh4t3a]
05-20 09:10:34:867 0xb0047a0 _csync_detect_update: file: VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF, instruction: INSTRUCTION_NEW <<=
05-20 09:10:34:867 0xb0047a0 csync_ftw: URI without fuzz for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF is "VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF"
05-20 09:10:34:867 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF Calling into main thread...
05-20 09:10:34:880 0x3feb28 OCC::AbstractNetworkJob::start: !!! OCC::LsColJob created for "https://files.srk.com/owncloud" + "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF"
05-20 09:10:34:918 0x3feb28 OCC::AbstractNetworkJob::slotFinished: void OCC::AbstractNetworkJob::slotFinished() 2 "Connection closed"
05-20 09:10:34:919 0x3feb28 OCC::DiscoverySingleDirectoryJob::lsJobFinishedWithErrorSlot: void OCC::DiscoverySingleDirectoryJob::lsJobFinishedWithErrorSlot(QNetworkReply*) "Connection closed" 0 2
05-20 09:10:34:920 0x3feb28 OCC::DiscoveryMainThread::singleDirectoryJobFinishedWithErrorSlot: void OCC::DiscoveryMainThread::singleDirectoryJobFinishedWithErrorSlot(int, QString) 5 "Connection closed"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) OCC::DiscoveryJob(0xa823b10) VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF ...Returned from main thread
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_opendir_hook: static csync_vio_handle_t* OCC::DiscoveryJob::remote_vio_opendir_hook(const char*, void*) 5 when opening VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF msg= "Connection closed"
05-20 09:10:34:921 0xb0047a0 csync_ftw: opendir failed for ownclouds://files.srk.com/owncloud/remote.php/webdav/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]/060_Final PDF - errno 5
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables/[Name of Deliverable3]"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design/!080_Deliverables"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River/1CC046.000_Tailings_Design"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES/Bul_River"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects/01_SITES"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) "/VAN Projects"
05-20 09:10:34:921 0xb0047a0 OCC::DiscoveryJob::remote_vio_closedir_hook: static void OCC::DiscoveryJob::remote_vio_closedir_hook(csync_vio_handle_t*, void*) OCC::DiscoveryJob(0xa823b10) ""
05-20 09:10:34:922 0xb0047a0 csync_statedb_close: sqlite3_close=0
```

**Comment by danimo** [comment]
@guruz @ogoffart This is probably unrelated to WND, and might "just" be our "connection closed" problem. right?

**Comment by bboule** [comment]
Hey Guys just checking on this as I am not sure where we stand?  Is there additional information I can provide?

**Comment by danimo** [comment]
After some discussion we agreed that this is most probably WND related, but a duplicate of https://github.com/owncloud/client/issues/2689. Please track there.
